docs/06_how-to-guides/30_key-value-api/kv_table/20_how-to-create-indexes-kv-table.md

```diff
@@ -59,7 +59,7 @@ class [[eosio::contract]] smrtcontract : public contract {
 1. Use the `KV_NAMED_INDEX` macro with two parameters.
```

Comment:
Is this how to missing the Overview section?

Reply:
no, it has it

```
## Overview

This how-to provides instructions to create the following indexes on a `Key-Value Table` (`kv table`):
```